net: lib: downloader: avoid continous loop on error event (#22334)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Libraries for networking
------------------------

* :ref:`lib_downloader` library:

>   * Fixed a bug in the shell implementation causing endless download retries on errors.
>   * Fixed a bug in the shell to allow multiple downloads.

Reviewer: ```suggestion
  * Fixed:
  
    * A bug in the shell implementation causing endless download retries on errors.
    * A bug in the shell to allow multiple downloads.
```

Libraries for networking
------------------------

* :ref:`lib_downloader` library:

>   * Fixed a bug in the shell implementation causing endless download retries on errors.

Reviewer 2: Add a note about fixing multiple downloads.

>   * Fixed a bug in the shell implementation where the download would be retried indefinitely on errors.

Reviewer: ```suggestion
  * Fixed a bug in the shell implementation causing endless download retries on errors.
```
